financial-connections/src/main/java/com/stripe/android/financialconnections/features/consent/ConsentScreen.kt

```diff
@@ -182,6 +184,11 @@ private fun ConsentMainContent(
                         bottom = 24.dp
                     )
             ) {
+                PhoneNumberCollectionSection(
```

Comment:
is this newly added? should we move it into a separate PR?

Reply:
Sorry this is still a draft, I was testing how it looks in Connections : )

Reply:
removed!